data/actionsmap/yunohost.yml

```
@@ -240,132 +250,82 @@ user:
                             nargs: "*"
                             metavar: USERNAME
                             extra:
                                 pattern: *pattern_username
                         -r:
-                            full: --remove-user
-                            help: User to remove in group
+                            full: --remove
+                            help: User(s) to remove in the group
                             nargs: "*"
                             metavar: USERNAME
                             extra:
                                 pattern: *pattern_username
 
                 ### user_group_info()
                 info:
-                    action_help: Get group information
+                    action_help: Get information for a specific group
                     api: GET /users/groups/<groupname>
                     arguments:
                         groupname:
-                            help: Groupname to get information
+                            help: Name of the group to get info about
                             extra:
                                 pattern: *pattern_username
 
         permission:
-            subcategory_help: Manage user permission
+            subcategory_help: Manage permissions
             actions:
+
                 ### user_permission_list()
                 list:
-                    action_help: List access to user and group
-                    api: GET /users/permissions/<app>
+                    action_help: List permissions and corresponding accesses
+                    api: GET /users/permissions/<permission>
                     arguments:
-                        -a:
-                            full: --app
-                            help: Application to manage the permission
-                            nargs: "*"
-                            metavar: APP
-                        -p:
-                            full: --permission
-                            help: Name of permission (main by default)
-                            nargs: "*"
-                            metavar: PERMISSION
-                        -u:
-                            full: --username
-                            help: Username
-                            nargs: "*"
-                            metavar: USER
-                        -g:
-                            full: --group
-                            help: Group name
-                            nargs: "*"
-                            metavar: GROUP
+                        -s:
+                            full: --short
+                            help: List only the names of permissions
+                            action: store_true
+                        -f:
+                            full: --full
+                            help: Display all informations known about each permissions, including the full list of users corresponding to allowed groups.
```

```suggestion
                            help: Display all info known about each permission, including the full user list of each group it is granted to.
```
Not 100% sure what the original is conveying.